CRM/Core/BAO/CustomGroup.php

```diff
@@ -1259,6 +1259,10 @@ public static function getAllCustomGroupsByBaseEntity($entityType) {
    */
   private static function _addWhereAdd(&$customGroupDAO, $entityType, $entityID = NULL, $allSubtypes = FALSE) {
     $addSubtypeClause = FALSE;
+    // This function isn't really accessible with user data but since the string
+    // is not passed as a param to the query adding slashes seems like a harmless
+    // precaution.
+    $entityType = addslashes($entityType);
```

Comment:
should we actually use `CRM_Core_DAO::escapeString` instead?

Reply:
fixed